### ISSUE TITLE
Can not build spine_flutter in release

### ISSUE THREAD

**Issue by KurtLa** [EsotericSoftware/spine-runtimes#2553]
When I try to build my application with spine_flutter in `release mode`, I get a `namespace error`:
```

* What went wrong:
A problem occurred configuring project ':spine_flutter'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 822ms
Running Gradle task 'bundleRelease'...                           1,308ms
Gradle task bundleRelease failed with exit code 1
```

**Comment by badlogic** [comment]
I'm afraid this is not enough information to help. Can you share the project? Please also run `flutter doctor` and post the output.

**Comment by KurtLa** [comment]
Hi @badlogic 
sure. I think the problem occurs because i'm using gradle >8.3.

[https://d.android.com/r/tools/upgrade-assistant/set-namespace](https://d.android.com/r/tools/upgrade-assistant/set-namespace)

How to reproduce:
- flutter create spine_test
- upgrade gradle to 8.3.1 via android studio
- flutter pub add spine_flutter
- flutter build aab  

Thats a sample project with upgraded gradle:
[spine_test.zip](https://github.com/user-attachments/files/15883116/spine_test.zip)


```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.22.0, on macOS 14.4.1 23E224 darwin-arm64, locale
    en-DE)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 15.2)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2023.2)
[✓] VS Code (version 1.90.0)
[✓] Connected device (3 available)
[✓] Network resources

• No issues found!
```

**Comment by badlogic** [comment]
Thanks! I've fixed this in spine-flutter release 4.2.29, which should be live in about 10 minutes from now.
